Species/Languages.md

```diff
@@ -1,27 +1,117 @@
+* **Chardonian**: A number of mutually understandable related languages and dialects spoken in the Chardonian Empire, but with some loan words from Northros languages. Self-consciously seen as the inheritors of the Drankorian language, and sometimes has similar sounds. Tends to use Latin or Italian, with the occasional "classical" Greek. 
+* **Cymean**: A number of mutually understandable related languages and dialects spoken in the Cymean peninsula. Cymean words are usually based on Spanish or Portugese, but with some Italian. It is usually understood by speakers of Isinguese. 
+* **Isinguese:** The language spoken in the now destroyed lands of the Istabor Alliance, but still spoken in some of the refugee communities including the [[Duchy of Maseau]] and the [[Refounded Alliance of Aurbez]]. Based on Spanish or French, and is usually understood by speakers of Sembaran (especially southern Sembarans) or Cymean. 
+* **Illorian**: The language spoken on the islands of Illoria, it is often understandable to Chardonian speakers, especially those from the [[Apporia|Apporian Peninsula]] or [[Chardon]] itself. No defined real world mapping yet.
+* **~Northern Dialects~**: A set of now-almost extinct languages spoken before the Great War in a number of the northern realms, such [[Volganis]], [[Amani]], the [[Ausberg Empire]], and the [[Republic of Varjatta]]. These languages were usually understood by speakers of Chardonian, and are based on Italian.
+
+%% Background Note:
+This is likely a complete list of Drankorian Family languages, although most of these languages are best thought of as collections of mutually comprehensible dialects.
+%%
+## Northros Language Family
+These languages derive from the ancient tongues that were spoken by the first human migrants from [[Hkar]]. They use a diverse set of alphabets.
+
+_Real world analog: Northros languages are based on Semitic languages_
+
+* **Mawar**:
+* **Deno'qai**
+* **Old Zimkovan**:
+* **Valgardi**: A language spoken by the [[Vargaldi]], although with many loan words from Eastros languages. Written using the ~Unnamed Mariner Language~ alphabet. No specific defined real world analog, although many Varlgardi names are Slavic due to the influence of the Eastros language families. 
+
+%% Background Note:
+There are numerous undeveloped cultures and partially developed cultures that likely speak a Northros language. This list should not be considered complete
+%%
+
+## Eastros Language Family
+All languages in the Eastros language family derive from ~Unnamed Mariner Language~, the language spoken by the ancient seafaring culture that originally settled the eastern parts of the Green Sea. They use the ~Unnamed Mariner Language~ alphabet.
+
```

Comment:
I think that Tollish and Sembaran should use the same alphabet, as that is a fairly big barrier. 

My head canon but doesn't have to be real canon is that a lot of the westernmost Eastros groups had an older alphabet replaced by Drankorian, perhaps vaguely paralleling the replacement of Anglo-Saxon runes with the latin alphabet.

So probably Tollish and maybe Vosic (not sure about that name?) now use the Drankorian alphabet, maybe others as well? But probably not Skaegish as they should feel like an old and distinctive language.

Reply:
yeah this part I already changed

Reply:
yeah this part I already changed - I agree that Tollish is defintely drankorian. Not sure about Vosic, in my head Skaegish and Vosic are more similar than Tollish and Vosic.